CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [0.0.4] - Pre-release
+
+- Ensure keyword escaping works in all contexts
```

Comment:
We had a subtitle like 'changed' in the other releases, maybe keep that pattern?